core/templates/pages/contributor-dashboard-page/contributor-dashboard-page.component.html

```diff
@@ -15,91 +15,107 @@
     <div ng-class="{'oppia-translate-y': $ctrl.defaultHeaderVisible, 'oppia-contributor-dashboard-header': !$ctrl.defaultHeaderVisible}">
       <div class="oppia-opportunities-tab-customization-container">
         <div class="oppia-opportunities-tab-header" ng-if="$ctrl.activeTabName === 'myContributionTab'">
-          <div class="oppia-profile-details-container">
-            <span ng-if="$ctrl.profilePictureDataUrl">
-              <img ng-src="<[$ctrl.profilePictureDataUrl]>" class="rounded-circle" alt="User Avatar">
-            </span>
-            <span ng-if="!$ctrl.profilePictureDataUrl">
-              <i class="material-icons oppia-profile-picture-icon">&#xE853;</i>
-            </span>
-          </div>
-          <div class="oppia-contributor-username-and-review-rights-container">
-            <div class="oppia-contributor-username-container" data-tab-name="username">
-              <lazy-loading ng-show="$ctrl.userInfoIsLoading"></lazy-loading>
-              <span class="protractor-test-username" ng-show="!$ctrl.userInfoIsLoading">
-                <[$ctrl.userIsLoggedIn ? $ctrl.username : 'Guest']>
-              </span>
-            </div>
-            <div class="oppia-review-right-details-container protractor-test-review-rights" ng-if="$ctrl.userIsReviewer">
-              <strong>Review rights:</strong>
-              <div ng-if="$ctrl.userCanReviewTranslationSuggestionsInLanguages.length > 0">
-                Translations in languages:
-                <span ng-repeat="languageDescription in $ctrl.userCanReviewTranslationSuggestionsInLanguages">
-                  <span class="oppia-review-item-detail-container protractor-test-translation-<[$ctrl.provideLanguageForProtractorClass(languageDescription)]>-reviewer"><[languageDescription]></span>
+          <div class="outer-container">
```

Comment:
I think we should prepend all our style classes with `oppia`

Reply:
Done.